Frameworks/CoreFoundation/URL.subproj/CFURL.c

```diff
@@ -5007,21 +5007,35 @@ CFURLRef CFURLCreateCopyDeletingPathExtension(CFAllocatorRef allocator, CFURLRef
     }
 #endif
     
+    CFStringRef lastPathComponent = NULL;
```

Comment:
*Any* change to cf not in the swift version should be documented with a // WINOBJC comment to make it easy to identify where we are diverging and why.